data_processor.py

```diff
@@ -0,0 +1,101 @@
+# title           : data_processor.py
+# description     : AWS Lambda script;
+#                   processes manual uploads to manual ingest bucket
+# author          : Volpe Center (https://www.volpe.dot.gov/)
+# license         : MIT license
+# ==============================================================================
+#
+
+from __future__ import print_function
+
+import logging
+import boto3
+import urllib
+import os
+import time
+import json
+from datetime import datetime
+
+CHUNK_SIZE = 1073741824 # 1 GB
+
+logger = logging.getLogger()

```

Comment:
I don't know of any advantages or disadvantages of printing vs logging...

Reply:
The logger utility I've seen is just slightly formatted printing....

Reply:
I'd prefer to do it a consistent way, but otherwise no strong opinion